helm/formatting_guidelines.md

```diff
@@ -0,0 +1,59 @@
+# Helm Chart Formatting Guidelines
+
+## Templating
+
+When using helm templating calls we include double spaces inside the double brackets
+like this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-configmap
+data:
+  myvalue: "Hello World"
+```
+
+In comparison to this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  # No spaces makes it harder to see the templating call
+  name: {{.Release.Name}}-configmap
+data:
+  myvalue: "Hello World"
+```
+
+## Indentation
+
+The indentation of our helm chart yaml files should be 2 spaces.
+Moreover we do not indent yaml lists, hyphens of the list members act as indentation.
```

Comment:
Also we should mention what quotation style we use. I guess double quotes, e.g. https://docs.helm.sh/developing_charts/#values-files

Reply:
Yes good call, unfortunatly I already merged it. But your welcome to open another PR :)